cli/src/commands/get/comments.rs

```diff
@@ -179,14 +212,61 @@ pub fn get_many(client: &Client, args: &GetManyCommentsArgs) -> Result<()> {
     }
 }
 
-#[derive(Default)]
+fn get_label_attribute_filter(
+    client: &Client,
+    dataset_id: DatasetIdentifier,
+    filter: &Regex,
+) -> Result<Option<AttributeFilter>> {
+    let dataset = client.get_dataset(dataset_id)?;
+
+    let label_names: Vec<LabelName> = dataset
+        .label_defs
+        .into_iter()
+        .filter(|label_def| filter.is_match(&label_def.name.0))
+        .map(|label_def| label_def.name)
+        .collect();
+
+    if label_names.is_empty() {
+        info!("No label names matching the filter '{}'", filter);
+        Ok(None)
+    } else {
+        info!(
+            "Filtering on label(s):\n- {}",
+            label_names
+                .iter()
+                .map(|label_name| label_name.0.clone())
+                .collect::<Vec<String>>()
+                .join("\n- ")
```

Comment:
Don't need to clone here with
```suggestion
            label_names
                .iter()
                .map(|label_name| label_name.0.as_str())
                .collect::<Vec<_>>()
                .join("\n- ")
```